Modules/modFileEncoding.ACM

```diff
@@ -3,6 +3,41 @@ Option Explicit
 
 Private Declare Sub CopyMemory Lib "kernel32" Alias "RtlMoveMemory" (ByVal Destination As Any, ByVal Source As Any, ByVal Length As Long)
 
+Function canConvUCS2ToANSI(strFilename As String) As Boolean

```

Comment:
This function does not appear to be used, maybe remove it?
`ConvFileUCS2_to_ANSI` has something very similar inlined.

Reply:
Agreed, removed